aspnetcore/fundamentals/servers/aspnet-core-module/sample/StartupEF.cs

```diff
@@ -0,0 +1,97 @@
+﻿#region snippet_Usings
```

Comment:
Why do you need an EF sample for ANCM? You don't show any dependency between the two.